tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -136,6 +149,142 @@ ZTEST_F(ascs_test_suite, test_sink_ase_read_state_idle)
 	zassert_equal(0x00, hdr.ase_state, "unexpected ASE_State 0x%02x", hdr.ase_state);
 }
 
+ZTEST_F(ascs_test_suite, test_cb_register_without_ascs_registered)
+{
+	int err;
+
+	/* Unregister ASCS, as its registered through setup */
+	bt_bap_unicast_server_unregister();
```

Comment:
Should add error checks for the unregister as well (applies to all cases in this file)